doc/admin-guide/files/records.config.en.rst

```diff
@@ -1326,6 +1326,53 @@ HTTP Redirection
    This setting determines the maximum size in bytes of uploaded content to be
    buffered for HTTP methods such as POST and PUT.
 
+.. ts:cv:: CONFIG proxy.config.http.redirect.actions STRING routable:follow
+   :reloadable:
+
+   This setting determines how redirects should be handled. The setting consists
+   of a comma-separated list of key-value pairs, where the keys are named IP
+   address ranges and the values are actions.
+
+   The following are valid keys:
+
+   ============= ===============================================================
+   Key           Description
+   ============= ===============================================================
+   ``loopback``  IPv4 ``127.0.0.0/8`` and IPv6 ``::1``
+   ``private``   IPv4 ``10.0.0.0/8`` ``100.64.0.0/10`` ``172.16.0.0/12`` ``192.168.0.0/16`` and IPv6 ``fc00::/7``
+   ``multicast`` IPv4 ``224.0.0.0/4`` and IPv6 ``ff00::/8``
+   ``linklocal`` IPv4 ``169.254.0.0/16`` and IPv6 ``fe80::/10``
+   ``self``      Addresses on which |TS| listens
+   ``routable``  All publicly routable addresses
+   ``default``   All address ranges not configured specifically
+   ============= ===============================================================
+
+   The following are valid values:
+
+   ========== ==================================================================
+   Value      Description
+   ========== ==================================================================
+   ``return`` Do not process the redirect, send it as the proxy response.
+   ``reject`` Do not process the redirect, send a 403 as the proxy response.
+   ``follow`` Internally follow the redirect up to :ts:cv:`proxy.config.http.number_of_redirections`. **Use this setting with caution!**
+   ========== ==================================================================
+
+   .. warning:: Following a redirect to other than ``routable`` addresses can be
+      dangerous, as it allows the controller of an origin to arrange a probe the
+      |TS| host. Generally, an administrator should never configure |TS| to
+      follow these redirects.
```

Comment:
* Is this the suggested rewording?`Generally, an administrator should never configure |TS| to follow these redirects for a publicly accessible host`

* For organizations with a zero trust model, lateral attacks from within the network could still pose a risk here. I am fine with rewording this how we would like. If we do say something here I was thinking it would be good not to be overly broad in the cases when this would seem acceptable.

Reply:
Maybe "enabling these redirects makes |TS| open to third party attacks and probing and therefore should only be considered in known safe environments".

Reply:
Sounds good, will update it.